Formula/restview.rb

```diff
@@ -0,0 +1,64 @@
+class Restview < Formula
+  include Language::Python::Virtualenv
+
+  desc "Viewer for ReStructuredText documents that renders them on the fly"
+  homepage "https://github.com/mgedmin/restview"
+  url "https://github.com/mgedmin/restview/archive/2.7.0.tar.gz"
+  sha256 "ab288234da5c6b3f7cd02d6112d0a020822f43423c086089efeeda2ffac5c388"
+
+  head "https://github.com/mgedmin/restview.git", :shallow => false
```

Comment:
What's the failure if you use a shallow clone?

Reply:
None – I just copied the line from another file. I've updated this.